packages/block-editor/package.json

```diff
@@ -28,6 +28,7 @@
 		"@wordpress/blocks": "file:../blocks",
 		"@wordpress/components": "file:../components",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "^2.4.0",
```

Comment:
block editor shouldn't depend on core-data we should instead refactor it to remove the need for this dependency.

Reply:
@jorgefilipecosta do you know why we still need this?

Reply:
Could we get at least `react-autosize-textarea` and `traverse` bits in independently, then?

Reply:
I agree that block-editor should not depend on core-data, and I don't know the reason we have this dependency. Maybe we should just remove the import from: https://github.com/WordPress/gutenberg//blob/a3ef9605c74a8f3e6cb57a28fad33e4701585d6a/packages/block-editor/src/index.js#L5

In my tests, it seems to work fine.

Reply:
@jorgefilipecosta can you help to remove it in another PR? :-)

Reply:
Hi @simison, I'm sorry for the delay I had some AFK time. I added this issue to my to-do list.

Reply:
It looks like @gziolo solved this problem in PR https://github.com/WordPress/gutenberg/pull/16969.

Reply:
This and way much more of issues :)